src/test/daml/Daml/Finance/Instrument/Bond/Test/Util.daml

```diff
@@ -121,12 +129,12 @@ lifecycleBond readAs today bondInstrument settler issuer observableCids = do
 
 verifyNoLifecycleEffects : [Party] -> Date -> Instrument.K -> Party -> Party -> [ContractId Observable.I] -> Script ()
 verifyNoLifecycleEffects readAs today bondInstrument settler issuer observableCids = do
```

Comment:
Rename parameters and variables here and in the functions below to remove the `bond` prefixes.

Reply:
I removed the bond references in this function. I also clarified that this is a general function that can be used for both swaps and bonds, and I clarified which of the other utility functions are general / bond only / swap only.